src/irr/asset/CPLYMeshFileLoader.cpp

```diff
@@ -597,17 +592,41 @@ bool CPLYMeshFileLoader::genVertBuffersForMBuffer(asset::ICPUMeshBuffer* _mbuf,
 
 	const uint32_t vertexSize = std::accumulate(sizes, sizes + 4, static_cast<size_t>(0));
 
+	// TODO default shader (uber shader isn't a goal since there will be only one vertex and fragment shader)
+	auto pushConstantRange = core::make_refctd_dynamic_array<core::smart_refctd_dynamic_array<SPushConstantRange>>(2);
+	pushConstantRange->operator[](0).stageFlags = ISpecializedShader::E_SHADER_STAGE::ESS_VERTEX;
+	pushConstantRange->operator[](0).offset = 0; //TODO
+	pushConstantRange->operator[](0).size = 0; //TODO
+
+	pushConstantRange->operator[](1).stageFlags = ISpecializedShader::E_SHADER_STAGE::ESS_FRAGMENT;
+	pushConstantRange->operator[](1).offset = 0; //TODO
+	pushConstantRange->operator[](1).size = 0; //TODO
+
+	auto mbPipelineLayout = core::make_smart_refctd_ptr<ICPUPipelineLayout>(pushConstantRange->begin(), pushConstantRange->end()); 
+
+	auto inputParams = core::make_smart_refctd_ptr<SVertexInputParams>();
+	inputParams->enabledAttribFlags = E_POS | E_COL | E_UV | E_NORM;
```

Comment:
nope, you need bit shifts to build the mask

Reply:
maybe we should do a template function with variadic arguments to handle bitfields creation at compile time? like `(1 << sth) | (1 << sth2) | etc`

Reply:
yes, why not, should go into one of the core headers then

Reply:
okay, so I will provide such